js/data/cryptoCurrencies.js

```diff
@@ -59,7 +59,16 @@ const currencies = [
     maxDisplayDecimals: 8,
     averageModeratedTransactionSize: 184,
     feeBumpTransactionSize: 154,
-    qrCodeText: address => address,
+    qrCodeText: address => {
+      let prefixedAddress = address;
+
+      if (address.split(':').length === 1) {
+        const prefix = app.serverConfig.testnet ? 'bchtest' : 'bitcoincash';
+        prefixedAddress = `${prefix}:${address}`;
+      }
+
+      return prefixedAddress;
+    },
```

Comment:
would using startsWith() be simpler here?

```
const prefix = app.serverConfig.testnet ? 'bchtest' : 'bitcoincash';
const prefixedAddress = address.startsWith(prefix) ?
  prefixedAddress : `${prefix}:${address}`;
```

Not a huge deal if you prefer the other way, assuming BCH addresses will never have a colon in them later on (which I think is a correct assumption).

FWIW, it's pretty standard in JS from what I've seen to do:

`if (address.split(':').length) {`
-instead of-
`if (address.split(':').length === 1) {`

-and-

`if (!address.split(':').length) {`
-instead of-
`if (address.split(':').length === 0) {`

Reply:
Agreed. Fixed and ready for merge.